src/v3.rs

```diff
@@ -262,22 +264,33 @@ impl Content {
     }
 
     /// Set the type of this content.
-    pub fn set_content_type(mut self, content_type: &str) -> Content {
-        self.content_type = String::from(content_type);
+    pub fn set_content_type<S: Into<String>>(mut self, content_type: S) -> Content {
+        self.content_type = content_type.into();
         self
     }
 
     /// Set the corresponding message for this content.
-    pub fn set_value(mut self, value: &str) -> Content {
-        self.value = String::from(value);
+    pub fn set_value<S: Into<String>>(mut self, value: S) -> Content {
+        self.value = value.into();
         self
     }
 }
 
 impl Personalization {
-    /// Construct a new personalization block for this message.
-    pub fn new() -> Personalization {
-        Personalization::default()
+    /// Construct a new personalization block for this message with to as vec containing one element, the email provided
```

Comment:
We don't need to necessarily comment on the implementation details regarding the vector. Perhaps just add a note about having the email be the "To" address.